content/_index.md

```diff
@@ -13,12 +13,39 @@ by Mozilla.
 
 [1]: https://www.rust-lang.org/
 
-## How to participate?
+## Participate
 
-- Join the [Finland Rust-lang Group][2] Meetup group and come to our meetups
-- Chat with us on the [#rust-finland:matrix.org][3] channel
-- Check out our Github repositories under the [rust-finland][4] organization
+### Meetups
+
+Join the [Finland Rust-lang Group][2] Meetup group and come to our meetups. We
+try to host meetups regularly with a variety of topics ranging from basic to
+advanced topics.
+
+No worries if you have no Rust experience or are just starting out. We welcome
+folks from all experience and interest levels.
+
+### Chat
+
+Chat with us on the [#rust-finland:matrix.org][3] channel. We talk news, Rust
```

Comment:
At least for me this link is broken in both Firefox and Chrome. I can't click on it. It renders as link though. ¯\_(ツ)_/¯

Reply:
You're right, not working on Ubuntu Chromium either. I wonder if it could be related to the `#` sign in front of the link text malforming the link conversion?

Reply:
Aaaa wait, my hacky way of preventing anchor linked headings from being hidden by a fixed header is rendering on top of it preventing all click events. Need to fix this.